azkaban-web-server/src/main/java/azkaban/webapp/WebServerProvider.java

```diff
@@ -61,12 +65,42 @@ public Server get() {
 
     // setting stats configuration for connectors
     setStatsOnConnectors(server);
+    disableHttpMethods(server);
 
     logger.info(String.format(
         "Starting %sserver on port: %d # Max threads: %d", useSsl ? "SSL " : "", port, maxThreads));
     return server;
   }
 
+    /**
```

Comment:
please format the change appropriately with save plugin https://github.com/azkaban/azkaban/blob/master/CONTRIBUTING.md